build-logic/build.gradle.kts

```diff
@@ -6,8 +6,17 @@ repositories {
     gradlePluginPortal()
```

Comment:
Comment for the `build-logic` directory: this build-logic pattern will be obvious for people who are proficient with Gradle, but not everyone is, unfortunately. 

Can you please create a `README.md` in the root of the `build-logic` directory and add some description for why this module exists and what it does? It would help with making Dokka beginner and contributor friendly, and will hopefully help the future generations of maintainers.

The text is up to you, but here are the things worth mentioning in my opinion:

* State that it allows to encapsulate complex build logic that also can be shared between projects, similar to [buildSrc](https://docs.gradle.org/7.6/userguide/organizing_gradle_projects.html#sec:build_sources).
* State that `build-logic` is a common pattern, and it does not cause the whole project to become out of date when changes are made, as opposed to `buildSrc`. Attaching a link to an article would also be useful, for instance https://proandroiddev.com/stop-using-gradle-buildsrc-use-composite-builds-instead-3c38ac7a2ab3